test/Generator.V3.Tests/TestDtdlModels/Space.json

```diff
@@ -50,6 +50,15 @@
       "writable": true,
       "name": "capabilities"
     },
+    {
+      "@type": "Property",
+      "name": "geometry",
+      "writable": true,
+      "schema": {
+        "@type": "Array",
```

Comment:
For GeoJSON Geometry purposes, we don't want 1 array property; Geometry should be its own model and have 4 sub-types: point, line, polygon, multi-polygon. Then the space model can have a relationship to the geometry object (that could happen to be a polygon or a multi-polygon.

The other alternative is to use the generic Array as you do here, but still has to be the type of the coordinates, not the geometry (geometry should have a `type` property as well).

Whichever approach you use, it's important to see it in action in the unit tests before we can move forward.